F/FastJet_Julia_Wrapper/build_tarballs.jl

```diff
@@ -0,0 +1,45 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "FastJet_Julia_Wrapper"
+version = v"0.5.0"
+
+# Collection of sources required to build FastJet_Julia_Wrapper
+sources = [
+   GitSource("https://github.com/jstrube/FastJet_Julia_Wrapper.git", "1ed483a0449a5a7330e0d899492b5c5d2247d47a"; unpack_target="FastJet_Julia_Wrapper"),
+   FileSource("https://github.com/JuliaPackaging/JuliaBuilder/releases/download/v1.0.0-2/julia-1.0.0-x86_64-w64-mingw32.tar.gz", "9c58bc0873e52cf6c41108a7a2b100f68419478f10c6fe635197b1bf47eec64d"; unpack_target="x86_64-w64-mingw32"),
```

Comment:
Do we need all these Julia tarballs?  Only Linux and macOS are used.

Reply:
indeed, we don't.